android/modules/ui/src/java/ti/modules/titanium/ui/widget/listview/ListSectionProxy.java

```diff
@@ -902,9 +902,11 @@ public void populateViews(KrollDict data, TiBaseListViewItem cellContent, TiList
 		}
 
 		//process listItem properties
-		KrollDict listItemDiff = cellContent.getViewItem().generateDiffProperties(listItemProperties);
-		if (!listItemDiff.isEmpty()) {
-			listItem.processProperties(listItemDiff);
+		if (cellContent.getViewItem().getView() != null) {
+			KrollDict listItemDiff = cellContent.getViewItem().generateDiffProperties(listItemProperties);
+			if (!listItemDiff.isEmpty()) {
+				listItem.processProperties(listItemDiff);
+			}
```

Comment:
We need to get rid of the `getView()` part from the `null` check to fix the [TIMOB-26719](https://jira.appcelerator.org/browse/TIMOB-26719) issue.

Reply:
Yep, cherry-picked into this PR 👍